uk.co.compucorp.civicrm.tasksassignments/api/v3/Task.php

```diff
@@ -601,12 +601,12 @@ function civicrm_api3_task_getoptions($params) {
 
     foreach ($types as $type) {
       if (!$sequential) {
-        $result[$type['value']] = $type['name'];
+        $result[$type['value']] = $type['label'];
       }
       else {
         $result[] = array(
           'key' => $type['value'],
-          'value' => $type['name'],
+          'value' => $type['label'],
         );
```

Comment:
@ajesamson please mention on the ticket description why these changes to the Task and Document APIs were necessary and what parts of the system did you test to make sure this didn't break anything

Reply:
I have the ticket description. Kindly check again please.